src/main/java/org/jenkinsci/plugins/registry/notification/DockerHubTrigger.java

```diff
@@ -138,6 +147,7 @@ public String getDisplayName() {
 
         @Override
         public Trigger<?> newInstance(StaplerRequest req, JSONObject formData) throws FormException {
+            //TODO can't remember why we are doing it this convoluted way but it looks intentional
```

Comment:
I think you made this comment, @rsandell - not sure what to do with it.

Reply:
yepp, just leave it there :)